data/transactions/logic/eval_test.go

```diff
@@ -4134,6 +4134,22 @@ func TestDig(t *testing.T) {
 	testPanics(t, "int 3; int 2; int 1; dig 11; int 2; ==; return", 3)
 }
 
+func TestCover(t *testing.T) {
```

Comment:
I think you *might* have to run `make sanity` on the root folder (`go-algorand`) if you haven't done so already -- I took a quick look at the Travis error and apparently there might be formatting issues.

Reply:
It looks like the exact statements are: 
```
  gofmt -w /home/travis/build/[secure]/go-[secure]/data/transactions/logic/eval.go
  gofmt -w /home/travis/build/[secure]/go-[secure]/data/transactions/logic/evalStateful_test.go
```
but I think `make sanity` will do the same thing?

Reply:
ok I just did that, hopefully it passes